support/collectors/collectors.go

```diff
@@ -129,6 +129,7 @@ func GetTalosNodeCollectors(ctx context.Context, client *client.Client) ([]*Coll
 	base := []*Collector{
 		NewCollector("dmesg.log", dmesg),
 		NewCollector("controller-runtime.log", logs("controller-runtime", false)),
+		NewCollector("dns-resolve-cache.log", logs("dns-resolve-cache", false)),
```

Comment:
should we collect `syslogd` logs too?

Reply:
`syslogd` is a service, so it gets automatically collected, but this log doesn't have a service behind it.